### ISSUE TITLE
Hotkeys for Next and Previous

### ISSUE THREAD

**Issue by Zokoner** [Nibuja05/FastProxer#3]
Hi,
first of all, thank you very much for your Extension.
I've tried my self on it an couldn't get it to work.

Maybe it shouldn't be my first Extension with no knowledge whatsoever.

It isn't really an issue so sorry if i post this on the wrong place.

I've looked in to your code and added two tiny hotkeys if you want to implement it officaly.

In stream_script.js

"use strict";
const HOTKEY_NEXT_VIDEO = 78; // 'n' key
const HOTKEY_PREV_VIDEO = 80; // 'p' key
let STATUS = "None";
...
//rest of the code
...
stream_Init();

// Hotkey event listener
document.addEventListener("keydown", (event) => {
  // Next Video
  if (event.keyCode === HOTKEY_NEXT_VIDEO) {
    stream_NextButtonClick();
  }

  // Previous Video
  if (event.keyCode === HOTKEY_PREV_VIDEO) {
    stream_PrevButtonClick();
  }
});

Thanks for your work.

Best regards.

[stream_script.zip](https://github.com/Nibuja05/FastProxer/files/11898401/stream_script.zip)

**Comment by Nibuja05** [comment]
Hey!
First of all thank you for the suggestion. I'm glad someone actually uses this extension, as it only started out as a personal project of mine. May I ask, how you come across this repo?

Adding this functionality should be no problem. I will add it officially in the next days probably as well as settings for the hotkeys. Thanks for your input!

**Comment by Zokoner** [comment]
I have for nearly every streaming service some kind of extension to improve my expirence with it.
I wasn't really expecting to find something but i found your FastProxer Extension in the Opera Store.
It was on Version 1.0 with some missing features or bugs, so i thought to do the rest by myself.
And after trying and trying i was about to contact you for some questions, but then i saw the github link to this, and just like that most of my problems was taken care of exept for the missing hotkeys.

TL;DR: I found this via FastProxer Extension in the Opera Store.

**Comment by Nibuja05** [comment]
Oh cool,
yeah I added the extension to to opera store a while ago. The new version there will be up, once the moderators gave their ok.

I added a new release (**3.2**) to this repo already, which has the shortcut support. It also comes with new settings for changing the hotkeys, as well as a hotkey for "Cancel".

I would really appreciate your feedback on the update.

**Comment by Zokoner** [comment]
Hi,
sorry for my late reply. I wasn't really able to test the new version until now. 
But it seems that anything but the Cinema mode dosen't work anymore. The hotkey button also dosen't do anything.
Sadly i can't tell when it stopped working. I've tested the extension in Chrome and Opera GX.
